Tests/UnitTests/EdgePublicAPITests.swift

```diff
@@ -0,0 +1,212 @@
+//
+// Copyright 2021 Adobe. All rights reserved.
```

Comment:
This is an existing file which I moved from the FunctionalTest folder, and then added new test cases. I moved it from within Xcode and it appears Git treated it as a delete + new file instead of a rename.